[2024-06-19T09:36:23.213+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-19T09:36:23.243+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T03:50:00+00:00 [queued]>
[2024-06-19T09:36:23.257+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T03:50:00+00:00 [queued]>
[2024-06-19T09:36:23.257+0530] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-19T09:36:23.290+0530] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backup_task> on 2024-06-19 03:50:00+00:00
[2024-06-19T09:36:23.294+0530] {standard_task_runner.py:63} INFO - Started process 1929 to run task
[2024-06-19T09:36:23.300+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Backup_Database', 'backup_task', 'scheduled__2024-06-19T03:50:00+00:00', '--job-id', '72', '--raw', '--subdir', 'DAGS_FOLDER/basic.py', '--cfg-path', '/tmp/tmp5rege0hz']
[2024-06-19T09:36:23.303+0530] {standard_task_runner.py:91} INFO - Job 72: Subtask backup_task
[2024-06-19T09:36:23.380+0530] {task_command.py:426} INFO - Running <TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T03:50:00+00:00 [running]> on host FN-21052948.
[2024-06-19T09:36:23.496+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarika' AIRFLOW_CTX_DAG_ID='Backup_Database' AIRFLOW_CTX_TASK_ID='backup_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-19T03:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-19T03:50:00+00:00'
[2024-06-19T09:36:23.496+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-19T09:36:23.515+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-19T09:36:23.518+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/kiit/airflow/bashScripts/backup.sh ']
[2024-06-19T09:36:23.523+0530] {subprocess.py:86} INFO - Output:
[2024-06-19T09:36:23.539+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T09:36:23.550+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T09:36:23.621+0530] {subprocess.py:93} INFO - mysqldump: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T09:36:23.631+0530] {subprocess.py:93} INFO - mysqldump: Error: 'Access denied; you need (at least one of) the PROCESS privilege(s) for this operation' when trying to dump tablespaces
[2024-06-19T09:36:23.804+0530] {subprocess.py:93} INFO - Backup completed in 0 seconds
[2024-06-19T09:36:23.804+0530] {subprocess.py:93} INFO - Backup Success in location: /home/kiit/airflow/backup | Required Space: 0GB, Available Space: 245.31GB
[2024-06-19T09:36:23.804+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-19T09:36:23.805+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-19T09:36:23.833+0530] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Backup_Database, task_id=backup_task, run_id=scheduled__2024-06-19T03:50:00+00:00, execution_date=20240619T035000, start_date=20240619T040623, end_date=20240619T040623
[2024-06-19T09:36:23.874+0530] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-19T09:36:23.891+0530] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-19T09:36:23.899+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-19T09:39:31.444+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-19T09:39:31.463+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T03:50:00+00:00 [queued]>
[2024-06-19T09:39:31.469+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T03:50:00+00:00 [queued]>
[2024-06-19T09:39:31.470+0530] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-19T09:39:31.486+0530] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backup_task> on 2024-06-19 03:50:00+00:00
[2024-06-19T09:39:31.488+0530] {standard_task_runner.py:63} INFO - Started process 2185 to run task
[2024-06-19T09:39:31.491+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Backup_Database', 'backup_task', 'scheduled__2024-06-19T03:50:00+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/basic.py', '--cfg-path', '/tmp/tmprob7v3nt']
[2024-06-19T09:39:31.493+0530] {standard_task_runner.py:91} INFO - Job 76: Subtask backup_task
[2024-06-19T09:39:31.540+0530] {task_command.py:426} INFO - Running <TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T03:50:00+00:00 [running]> on host FN-21052948.
[2024-06-19T09:39:31.607+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarika' AIRFLOW_CTX_DAG_ID='Backup_Database' AIRFLOW_CTX_TASK_ID='backup_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-19T03:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-19T03:50:00+00:00'
[2024-06-19T09:39:31.608+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-19T09:39:31.617+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-19T09:39:31.617+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/kiit/airflow/bashScripts/backup.sh ']
[2024-06-19T09:39:31.621+0530] {subprocess.py:86} INFO - Output:
[2024-06-19T09:39:31.628+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T09:39:31.633+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T09:39:31.646+0530] {subprocess.py:93} INFO - mysqldump: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T09:39:31.651+0530] {subprocess.py:93} INFO - mysqldump: Error: 'Access denied; you need (at least one of) the PROCESS privilege(s) for this operation' when trying to dump tablespaces
[2024-06-19T09:39:31.828+0530] {subprocess.py:93} INFO - Backup completed in 0 seconds
[2024-06-19T09:39:31.828+0530] {subprocess.py:93} INFO - Backup Success in location: /home/kiit/airflow/backup | Required Space: 0GB, Available Space: 245.31GB
[2024-06-19T09:39:31.829+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-19T09:39:31.829+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-19T09:39:31.858+0530] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Backup_Database, task_id=backup_task, run_id=scheduled__2024-06-19T03:50:00+00:00, execution_date=20240619T035000, start_date=20240619T040931, end_date=20240619T040931
[2024-06-19T09:39:31.905+0530] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-19T09:39:31.926+0530] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-19T09:39:31.933+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
